src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -16,10 +16,10 @@
     private TaskRepository taskRepository;
     @Autowired
     private AdminConfig adminConfig;
-    private static final String SUBJECT = "Tasks: New Trello Card";
+    private static final String SUBJECT = "Tasks: Trello Cards";
 
     //@Scheduled(cron = "0 0 10 * * *")
-    //@Scheduled(fixedDelay = 10000)
+    @Scheduled(fixedDelay = 100000)
```

Comment:
100000 sekund, to nie jeden dzien jak w zadaniu? To jest prawdopodobnie do poprawy